.github/actions/action-npm-release/action.yml

```
@@ -0,0 +1,71 @@
+        echo "VERSION=$NEW_VERSION" >> "$GITHUB_OUTPUT"
+        git config --global user.email "github-actions@github.com"
+        git config --global user.name "GitHub Actions"
+        git add package.json package-lock.json
+        git commit -m "Bump version to ${NEW_VERSION}" || exit 1
+        git tag "${NEW_VERSION}" || exit 1
+        git push && git push --tags || exit 1
+      shell: bash
+    - id: node
+      uses: it-at-m/.github/.github/actions/action-npm-build@main
+      with:
+        app-path:  "${{ inputs.app-path }}"
```

YAMLlint (1.35.1)

[error] too many spaces after colon

(colons)